sdk/storage/Azure.Storage.Queues/src/QueueClient.cs

```diff
@@ -411,6 +417,352 @@ private void SetNameFieldsIfNull()
         }
         #endregion Create
 
+        #region Create If Not Exists
```

Comment:
NIT - remove spaces. Also I believe for blobs, I put the `CreateIfNotExists` methods in the `Create` region

Reply:
Fixed the spacing issue.